charters/wot-ig-2019.html

```diff
@@ -167,278 +179,202 @@ <h1 id="title">Proposed Web of Things Interest Group Charter</h1>
-            <dt id="wot-ucr" class="spec"><a href="#">WoT Use Cases and Requirements</a></dt>
-            <dd>
-              <p>This specification defines updated use cases and requirements for the Web of Things based on concrete scenarios of consumer, 
-                industrial, environmental, energy management and smart cities applications.</p>
-              <p class="draft-status"><b>Draft state:</b> <a href="#">Editor's Draft</a></p>
-              <p class="milestone"><b>Expected completion:</b> Q4 2020]</i></p>
-            </dd>
-
-            <dt id="wot-simple-api" class="spec">WoT Simple API</dt>
-            <dd>
-              <p>This specification outlines the requirements and a proposal for a simplified API for WoT application developers.</p>
-              <p class="draft-status"><b>Draft state:</b> No draft</p>
-              <p class="milestone"><b>Expected completion:</b> Q4 2020]</i></p>
-            </dd>
-
-            <dt id="wot-hypermedia" class="spec">WoT Hypermedia-driven Interactions</dt>
-            <dd>
-              <p>This specification outlines the requirements and a proposal for a hypermedia-based mechanism to drive machine clients through complex Thing Actions and Events.</p>
-              <p class="draft-status"><b>Draft state:</b> No draft</p>
-              <p class="milestone"><b>Expected completion:</b> Q4 2020]</i></p>
-            </dd>
-
-            <dt id="wot-profiles" class="spec">WoT Profiles for application and protocol domains</dt>
-            <dd>
-              <p>This specification outlines the requirements and a proposal for profiles for protocol and application .</p>
-              <p class="draft-status"><b>Draft state:</b> No draft</p>
-              <p class="milestone"><b>Expected completion:</b> Q4 2020]</i></p>
-            </dd>
-          </dl>
-        </div>
+        <section id="normative">
+          <h3>Normative Specifications</h3>
+          <p>
+            The Interest Group will not deliver any normative specifications.
+          </p>
+        </section>
 
-        <div id="ig-other-deliverables">
-          <h3>
-            Other Deliverables
-          </h3>
+        <section id="ig-other-deliverables">
+          <h3>Other Deliverables</h3>
           <p>
-            Other non-normative documents may be created based on the exploration work of the Interest Group. These may include:
-            <ul>
-              <li>WoT Infrastructure Services</li>
-              <li>WoT Lifecycle Management</li>
-              <li>WoT TD Profiles</li>
-            </ul>
+            Other non-normative documents may be created such as:
           </p>
-        </div>
+          <ul>
+            <li>use case and requirement documents;</li>
+            <li>experience and/or implementation reports for identified building blocks;</li>
+            <li>primer, best practice, or test case documents to support WoT developers when designing applications and/or Things.</li>
```

Comment:
Maybe add "including for security."   Although, leaving it out is probably also ok.